src/lib/OpenEXR/ImfFastHuf.cpp

```diff
@@ -11,6 +11,40 @@
 #include <string.h>
 #include <vector>
 
+// Static enabling/disabling the fast huffman decode
+
+
+#if defined(__clang__)
+//
+// Enabled for clang on Apple platforms (tested):
+//
+
+#    if defined(__APPLE__)
```

Comment:
This should also be supported by the [MSYS2 CLANGARM64 environment](https://www.msys2.org/docs/environments/), so could in theory change to just `__aarch64__`?

Reply:
We would not enable support on configurations we have not tested, in theory it could be done.